modules/@ergonode/attributes/src/pages/attribute-groups/_group/_id.vue

```diff
@@ -74,10 +74,11 @@ export default {
             __clearTranslationsStorage: '__clearStorage',
         }),
         onRemove() {
-            this.$openModal({
-                key: MODAL_TYPE.GLOBAL_CONFIRM_MODAL,
-                message: 'Are you sure you want to delete this group?',
-                confirmCallback: () => this.removeAttributeGroup({
+            this.$confirm({
+                type: MODAL_TYPE.DESTRUCTIVE,
+                title: 'Are you sure you want to delete this group?',
```

Comment:
to translate?

Reply:
I'll create other task to translate each module